### DISCUSSION TITLE
Speed for Direct DIA for phosphoproteome

### DISCUSSION THREAD

**Discussion by grazzilybear** [vdemichev/DiaNN#429] [Q&A]
We are generating the DIA phosphoproteome. However, it took more than one week to generate the predicted library and is not yet done. Should I do any specific setting to speed up the process? Thanks

**Comment by vdemichev** [comment]
Try only consider charges 2-3, only the precursor m/z range measured in your runs, no var mods except for phospho, max 3 var mods, max 1 missed cleavage. Then will be quicker.

**Reply by grazzilybear** [reply]
Because the phosphopeptides have higher misscvlg rate than regular peptides, I set up charges 2-3 and max 3 var mods, max 2 missed cleavage for direct-DIA. But I still can't get in-silico library even though I ran it for one week.......  Do you have any solution for this?
Thanks
